pkg/model/node_pool_config.go

```diff
@@ -75,6 +75,19 @@ func (c NodePoolConfig) FeatureGates() api.FeatureGates {
 	if gates == nil {
 		gates = api.FeatureGates{}
 	}
+	//From kube 1.11 PodPriority and ExpandPersistentVolumes have become enabled by default,
```

Comment:
Again, can we instead follow kubernetes lead and also enable them but allow us to disable them if desired through our configs?